2.7.0/todo/result_intel.md

```diff
@@ -30,12 +30,18 @@ The categories below are as follows:
 ### deprecation
 ### new features
 ### improvements
+- Upgrade submodule oneDNN to v3.7.1 ([#148293](https://github.com/pytorch/pytorch/pull/148293))
+- [CPU][Brgemm] add support for int8 brgemm ([#143384](https://github.com/pytorch/pytorch/pull/143384))
 ### bug fixes
-- Implement blend operation for float, double, int in VEC ATen backend for SVE ([#146479](https://github.com/pytorch/pytorch/pull/146479))
+- [Inductor] Use sleef implementation for CPP backend asinh codegen ([#142360](https://github.com/pytorch/pytorch/pull/142360))
```

Comment:
```suggestion
- Use sleef implementation for CPP backend `asinh` codegen (#142360)
```
Please backtick all code, also following the guidelines in doc above

Reply:
Thanks for the suggestion and changed.